example/src/usecases/Choir.js

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import { MultiModalComponent, SearchConfig, searchTypes } from 'trompa-multimodal-component';
+
+const config = new SearchConfig({
+  searchTypes: [searchTypes.AudioObject],
+  fixedFilter: { contributor_starts_with: 'https://www.voiceful.io' },
+});
+
+export default () => {
+  return (
+    <MultiModalComponent
+      config={config}
+      onResultClick={item => console.log(item)}
```

Comment:
Als het voor demo purposes is, dan is een alert misschien handiger voor non-developers?

Reply:
This is only for developers, it's the example project using the multimodal component which is only available by cloning the repository or by viewing the code via Github.